services/repository-managers/src/main/java/org/sagebionetworks/evaluation/manager/EvaluationQuotaUtil.java

```diff
@@ -0,0 +1,79 @@
+package org.sagebionetworks.evaluation.manager;
+
+import java.util.Date;
+
+import org.sagebionetworks.evaluation.model.Evaluation;
+import org.sagebionetworks.evaluation.model.SubmissionQuota;
+import org.sagebionetworks.repo.model.InvalidModelException;
+import org.sagebionetworks.util.Pair;
+
+public class EvaluationQuotaUtil {
```

Comment:
rename to SubmissionQuotaUtil (since you renamed EvaluationQuota to SubmissionQuota)